src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlCreation.cs

```diff
@@ -94,6 +94,12 @@ private static XElement CreateCompilationOptionsElement(CompilationOptions optio
                 element.SetAttributeValue(CheckOverflowAttributeName, true);
             }
 
+            if (options.OutputKind != OutputKind.DynamicallyLinkedLibrary)
```

Comment:
is there reason why you set this only when kind is not dynamically linked library? you could always set it? even though default is set to dynamicallylinked library?

Reply:
No strong reason. Since the default `OutputKind` is `DynamicallyLinkedLibrary`, I figured it's not useful to do more work in the common case.